music21/converter/subConverters.py

```diff
@@ -1019,8 +1031,13 @@ def write(self, obj, fmt, fp=None, subformats=None,
             defaults.title = ''
             defaults.author = ''
 
+        dataBytes: bytes = b''
         generalExporter = m21ToXml.GeneralObjectExporter(obj)
-        dataBytes: bytes = generalExporter.parse()
+        if makeNotation is False and 'Score' in obj.classes:
```

Comment:
Unclear to me why we should be searching for Score in classes -- if someone is using `makeNotation=False` they should know what they're doing and give a well-formed object.  This allows reverting the .parse(makeNotation=False) notes

Reply:
So, in that case, if someone does myPart.show(makeNotation=False), should we just raise MusicXMLExportException?

Reply:
I'm trying to understand what the motivation for this attribute is except "I'm a super power user working with 7,000 scores and I want to speed something up"  in which case I hope they'll realize that the user's guide is there to help.

I was going to ask in fact that the documentation for "makeNotation=False" says that it is a very advanced feature that assumes you know the ins and outs of music21 and musicxml fully before running it.  Just trying to save support time in the future.  (It's why I immediately close issues for "I wanted to use _coreInsert() because it should be faster, but I couldn't get it to work, can you help?")

Reply:
Ya, happy to add a caveat. I think the motivation was that makeNotation fiddles with beams, Tuplet brackets, accidentals etc., and so you can't necessarily depend on changes you made to the Python streams making it out the door. At least that's how I ended up here with tuplets long ago.

Reply:
That said, those hiccups may have been minimized after #1005 which included some of what was in an earlier version of this PR -- reading from actual streamStatus instead of streamStatus of a dummy stream never exposed to the user. But, we've also talked about adding things to makeNotation/export so still good to have, I think.